docs/platform-external/installing.md

```diff
@@ -0,0 +1,509 @@
+the provider offerings.
+
+### DNS
+
+There is no specific configuration for DNS when using platform external, the setup
+must follow the same as agnostic installation and the provider configuration.
+
+Please take a look at the following guides to the DNS setup:
+
+- [User-provisioned DNS requirements](https://docs.openshift.com/container-platform/4.13/installing/installing_platform_agnostic/installing-platform-agnostic.html#installation-dns-user-infra_installing-platform-agnostic)
+- [Validating DNS resolution for user-provisioned infrastructure](https://docs.openshift.com/container-platform/4.13/installing/installing_platform_agnostic/installing-platform-agnostic.html#installation-user-provisioned-validating-dns_installing-platform-agnostic)
+
+### Load Balancers
+
+The ["Load balancing requirements for user-provisioned infrastructure"](https://docs.openshift.com/container-platform/4.13/installing/installing_platform_agnostic/installing-platform-agnostic.html#installation-load-balancing-user-infra_installing-platform-agnostic).
+
+You can use the cloud provider's Load Balancer when it meets the requirements.
+
+Important notes:
+
+- the address `api-int.clusterDomain` must point to the internal Load Balancer address.
+- the Load Balancers must support hairpin connections
+
+## Preparing the installation
+
+The platform external configuration is created in this section. The steps
+below describe how to create the `install-config.yaml` with the required fields, and
+the steps used to customize the providers' manifests before generating the deployment/ignition
+configuration/ignition.
+
+### Create the install-config.yaml
+
+Follow the steps to [Manually create the installation configuration file](https://docs.openshift.com/container-platform/4.13/installing/installing_platform_agnostic/installing-platform-agnostic.html#installation-initializing-manual_installing-platform-agnostic),
+customizing the `platform` object, setting the type to `external`, and the
+`platformName` to the cloud provider's name:
+
+> TODO/check: is the installer's survey set correctly the platform external type in config? How the ProviderName is capture in the suvery stage?
+> Q: Do we need to add ./openshift-install create install-config? Or just provide a
+> full install-config.yaml example
+
+```yaml
+platform:
+  external:
+    platformName: "providerName"
+```
+
+### Create manifests
+
+Once the `install-config.yaml` was created, you can generate the manifests for the
+deployment by running the command:
+
+```bash
+./openshift-install create manifests
+```
+
+The `openshift/` and `manifests/` directory will be created.
+
+The steps below describe how to customize the OpenShift installation.
+
+The manifests directory will be copied to the bootstrap and applied when the core
+components are initialing the Control Plane.
+
+#### Create custom manifests for CCM
+
+In that stage, the Cloud Controller Manager (CCM) manifests must be created, as it is
```

Comment:
This might give the impression that a CCM is required. I suggest starting with something like "If a Cloud Controller Manager is available for the platform, it can be incorporated by..."

Reply:
thanks! Section revisited